ZXing.Net.Mobile/iOS/ZXingScannerView.ios.cs

```diff
@@ -620,7 +620,7 @@ public void Torch(bool on)
 				{
 					device.LockForConfiguration(out var err);
 
-					if (err != null)
+					if (err == null)
```

Comment:
Better to check the result of the operation instead:
```suggestion
                    if (device.LockForConfiguration(out var err))
```